Assets/HoloToolkit-Examples/InteractiveElements/Scripts/Controls/ButtonThemeWidget.cs

```diff
@@ -30,6 +31,9 @@ public class ButtonThemeWidget : InteractiveThemeWidget
         [Tooltip("scale animation component: optional")]
         public ScaleToValue ScaleSize;
 
+        [Tooltip("Upate the widget's theme and refresh if the theme tag changes")]
+        public bool WatchForThemeChange = false;
```

Comment:
nit: Would a default setting of true be preferred so that buttons automatically adopt theme changes? When would someone want this to be set to false? Would you want it to be false to save a little bit of performance in the Update method?